library/ecp.c

```diff
@@ -1254,6 +1261,7 @@ static inline int mbedtls_mpi_sub_mod( const mbedtls_ecp_group *grp,
 cleanup:
     return( ret );
 }
+#endif /* !(NO_FALLBACK && alt-implemented all referencing functions) */
```

Comment:
Instead of "alt-implemented all referencing functions" rather "all functions referencing mbedtls_mpi_sub_mod() are alt-implemented" ?

Reply:
Fixed